source/client-side-encryption/tests/legacy/fle2v2-CreateCollection-OldServer.yml

```diff
@@ -0,0 +1,41 @@
+# Requires libmongocrypt 1.8.0.
+runOn:
+  - minServerVersion: "6.0.0"
+    maxServerVersion: "6.3.99"
+    # FLE 2 Encrypted collections are not supported on standalone.
+    topology: [ "replicaset", "sharded", "load-balanced" ]
+    
+database_name: &database_name "default"
+collection_name: &collection_name "default"
```

Comment:
I noticed that various tests hard-code `collection: "encryptedCollection"` in operations. Would it make more sense for them to reference `*collection_name`?

Reply:
Not using the root `database_name` and `collection_name` was intended to avoid interacting with the drop/create behavior specified in the test runner.

https://github.com/mongodb/specifications/pull/1396/commits/1996d637689bc3b4612c5ddf275abc12ab8a9439 updates to use YAML anchors for the collection names.